test/models/task_test.rb

```diff
@@ -2,37 +2,75 @@
 
 describe Task do
   let(:porter_task) { build(:porter_job_results) }
+  let(:task) { Task.new }
 
-  let(:task) { Task.create }
+  describe ".callback" do
```

Comment:
Trying to more consistently write `describe ".class_method"` and `describe "#instance_method"` syntax.